Source/WebKit/Configurations/WebKit.xcconfig

```diff
@@ -190,9 +190,9 @@ OTHER_LDFLAGS = $(inherited) -iframework"$(SDK_DIR)$(SYSTEM_LIBRARY_DIR)/Private
 REEXPORTED_FRAMEWORK_NAMES = WebKitLegacy;
 REEXPORTED_FRAMEWORK_NAMES[sdk=macosx*] = $(inherited) WebCore;
 
-// FIXME: Remove -reexport_install_name once rdar://problem/30820233 is fixed.
+// FIXME: Remove -reexport_framework once rdar://problem/30820233 is fixed.
 OTHER_TAPI_FLAGS = $(OTHER_TAPI_FLAGS_$(WK_COCOA_TOUCH));
-OTHER_TAPI_FLAGS_cocoatouch = $(inherited) -reexport_install_name $(WK_ALTERNATE_WEBKIT_SDK_PATH)$(SYSTEM_LIBRARY_DIR)/PrivateFrameworks/WebKitLegacy.framework/WebKitLegacy -extra-private-header $(SRCROOT)/Platform/ExtraPrivateSymbolsForTAPI.h -extra-public-header $(SRCROOT)/Platform/ExtraPublicSymbolsForTAPI.h -exclude-private-header $(BUILT_PRODUCTS_DIR)/$(PRIVATE_HEADERS_FOLDER_PATH)/NSURLDownloadSPI.h;
+OTHER_TAPI_FLAGS_cocoatouch = $(inherited) -reexport_framework $(WK_ALTERNATE_WEBKIT_SDK_PATH)$(SYSTEM_LIBRARY_DIR)/PrivateFrameworks/WebKitLegacy.framework/WebKitLegacy -extra-private-header $(SRCROOT)/Platform/ExtraPrivateSymbolsForTAPI.h -extra-public-header $(SRCROOT)/Platform/ExtraPublicSymbolsForTAPI.h -exclude-private-header $(BUILT_PRODUCTS_DIR)/$(PRIVATE_HEADERS_FOLDER_PATH)/NSURLDownloadSPI.h;
```

Comment:
The ideal cleanup here would be to clean up the FIXME and delete the `-reexport` argument altogether. You can see from open-source build logs that the radar has been fixed. For example, in the build log for https://build.webkit.org/#/builders/661/builds/4787, we ran:

```
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/tapi installapi 
-verify-against /Volumes/Data/worker/Apple-iOS-16-Release-Build/build/WebKitBuild/Release-iphoneos/WebKit.framework/WebKit 
…
-reexport_framework WebKitLegacy 
…
-reexport_install_name /System/Library/PrivateFrameworks/WebKitLegacy.framework/WebKitLegacy 
-extra-private-header…
```

So the build system is passing `-reexport_framework` for us, and we can remove our manual workaround.

Reply:
Tried that and it resulted in build errors. Will try again though.

Reply:
If that doesn't work, I recommend we do not change this line. As far as I can tell, `-reexport_install_name` isn't deprecated (could you show me if I'm wrong?). And `-reexport_framework` takes a framework name, not a path:

```
> xcrun tapi installapi -help
…
  -reexport_framework <value>
                          Re-export the specified framework
```

Ultimately, this feels like something that isn't broken or deprecated, and shouldn't be changed, until we can resolve the FIXME.

Reply:
This FIXME is about using `REEXPORTED_FRAMEWORK_NAMES` instead of `OTHER_TAPI_FLAGS` for this. But I'm not sure if that's accurate, Apple engineers can consult rdar://61788689 for more details.